service/client.go

```diff
@@ -72,4 +72,14 @@ type Client interface {
 
 	// Returns whether we attached to a running process or not
 	AttachedToExistingProcess() bool
+
+	// Returns the address of a location expression
+	// loc ::= <filename>:<line> | <function>[:<line>] | /<regex>/
+	// * <filename> can be the full path of a file or just a suffix
+	// * <function> ::= <package>.<receiver type>.<name> | <package>.(*<receiver type>).<name> | <receiver type>.<name> | <package>.<name> | (*<receiver type>).<name> | <name>
+	// * <function> must be unambiguous
+	// * /<regex>/ will return a breakpoint for each function matched by regex
```

Comment:
s/breakpoint/location/

also

This is inconsistent with the info commands. Most of those accept a regexp but do not require /<regexp>/. I understand it's difficult to determine whether it's a regexp or not otherwise, but I don't like this inconsistency with other commands.